### PR TITLE
Release post and changelog

### PR DESCRIPTION
**Pull request by weiznich**
cc @Ten0 especially for the release blog entry (Roadmap, call for participation and stuff)

### CODE REVIEW

File: src/changelog.md [hunk 1]
```diff
@@ -0,0 +1,1951 @@
...
+
+All user visible changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/), as described
+for Rust libraries in [RFC #1105](https://github.com/rust-lang/rfcs/blob/master/text/1105-api-evolution.md)
+For any named minimal supported Rust version we guarantee that it is possible to build Diesel with the 
+default features enabled using some set of dependencies. Those set of dependencies is not necessarily 
+an up to date version of the specific dependency. We check this by using the unstable `-Z minimal-version` cargo flag. 
+Increasing the minimal supported Rust version will always be coupled at least with a minor release.
+
+## [2.0.0] 2022-07-07
```
Comment (weiznich) [inline src/changelog.md:21]: I just put some date here for now. We need to update that before the release.

File: src/news/2_0_0_release.md [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+
+The release of Diesel 2.0 does not only include the features listed above, but also marks the
+point where the following things can be provided by third party crates:
+
+* Custom `QueryDsl` extensions to support previously unsupported SQL features. Checkout 
+[`diesel_full_text_search`](https://github.com/diesel-rs/diesel_full_text_search) for an example
+* Alternative query dsl implementations reusing the existing `Connection` infrastructure
+* Custom [`Connection`](https://docs.diesel.rs/2.0.x/diesel/connection/trait.Connection.html#provide-a-new-connection-implementation-for-an-existing-backend) implementations for existing backends
+* Custom [`Connection`](https://docs.diesel.rs/2.0.x/diesel/connection/trait.Connection.html#implement-support-for-an-unsupported-database-system) 
+and [`Backend`](https://docs.diesel.rs/2.0.x/diesel/backend/trait.Backend.html#implementing-a-custom-backend) implementations for previously unsupported backends. Checkout [diesel-oci](https://docs.diesel.rs/2.0.x/diesel/backend/trait.Backend.html#implementing-a-custom-backend) for an example.
```
Comment (Ten0) [inline src/news/2_0_0_release.md:151]:
```suggestion
and [`Backend`](https://docs.diesel.rs/2.0.x/diesel/backend/trait.Backend.html#implementing-a-custom-backend) implementations for previously unsupported backends. Checkout [diesel-oci](https://github.com/GiGainfosystems/diesel-oci) for an example.
```

**Review by Ten0 — APPROVED** [review]
Release blog entry LGTM :)

File: src/news/index.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+include-after: |
+    <script src="../assets/javascripts/application.js"></script>
+---
+
+::: content-wrapper
+::: guide-wrapper
+
+#### [Diesel 2.0.0](./2_0_0_release.html)
+
+Diesel 2.0 introduces support for `GROUP BY` and `UNION`/`INTERSECT` queries. Additionally it features table aliases and a improved field mapping mechanism.
```
[the remark this answers is not in the document]
Comment (weiznich) [reply src/news/index.md:14]: https://github.com/sgrif/diesel.rs-website/pull/164/commits/31bea47c8b9ed7418e81b8b2f7451ffe820e2eee adds a note about this fix to the release blog.

**Pull request merged**